covenant.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Our Covenant | Fosstodon Hub
+description: The Fosstodon Team Covenant.
+permalink: /covenant/
+layout: default
+---
+## Fosstodon Team Covenant
+v0.1.0-2025-06-19
+
+The Fosstodon team is responsible for the upkeep of Fosstodon. While not all members of the team engage directly in all forms of moderation, all commit to the following:
+
+1. Acknowledge our privileges. Above all else, taking on responsibility for Fosstodon grants us privileges and authority that others do not have. This is a pledge to use our privilege for good.
+2. Aim for safety, prioritize those who are vulnerable, and consider impact before intentions. This includes believing people when their own actions show us who they are.
+3. Consider our potential for ignorance, and trust those who bring concerns. We will verify before acting, and seek relevant advice when we have doubts.
+4. Engage with respect, acknowledge the humanity of all involved, and take the minimum action required to have a positive impact on safety.
+5. Bear the burden of criticism, take accountability for mistakes, and commit to meaningful action and change.
```

Comment:
```suggestion
5. Bear the burden of criticism, be accountable for mistakes, and commit to meaningful action and change.
```

Reply:
There's a distinct difference here. "Take accountability" is active language and the activeness is meaningful. It implies that the accountability is welcomed. "Be accountable" is passive language and does not imply cooperation in the process.